dcs/unittype.py

```diff
@@ -84,8 +84,7 @@ class FlyingType(UnitType):
     property_defaults: Optional[Dict[str, Any]] = None
 
     pylons: Set[int] = set()
-    livery_name: Optional[str] = None
-    Liveries: LiverySet = LiverySet()
+    livery_name: str = ""
```

Comment:
This change may not be that obvious, but that's cause mypy complains about `livery_name` possibly being `None`, which violates the typing in `Liveries.__getitem__`. But looking back at this I wonder if it might break something w.r.t. mission generation. Any thoughts?

Reply:
theoretically it could break, as now it is expected that `livery_name` is always not None.
But I guess you should know best now, if this really has some impact?

Reply:
Afaik an empty string as livery will fall back to DCS' default. The `default_livery` method in unittype.py also returns an empty string in case it can't find one, so I would assume this isn't going to break things.

Despite that, it might not hurt to wait until I can confirm there's nothing that depends on `livery_name` being `None`.

Reply:
> an empty string as livery will fall back to DCS' default

This would make a great code comment.

Reply:
> > an empty string as livery will fall back to DCS' default
> 
> 
> 
> This would make a great code comment.

In general, any invalid livery should fall back to tet default. However I remember having a "missing textures" issue when testing the F-15D, so I think there's a caveat to it but I'd have to do some tests to double check.

Reply:
This is gonna sound funny 🤣 

So my last comment is totally irrelevant, fact is I'm the one who added to class variable in the first place, and it's not representing a livery name/id, but rather the name of the aircraft's livery folder, e.g. "A-10C II".

So yeah, this shouldn't be a problem at all. But in order to avoid confusion in the future, I'm gonna add a comment to it.